core/new-gui/src/app/workspace/component/workflow-editor/workflow-editor.component.spec.ts

```diff
@@ -27,6 +30,10 @@ class StubWorkflowActionService {
   public getJointGraphWrapper(): JointGraphWrapper {
     return this.jointGraphWrapper;
   }
+
+  public getTexeraGraph(): WorkflowGraphReadonly {
+    return this.texeraGraph;
+  }
```

Comment:
This method is not used anywhere in the code.

Reply:
I think it is used in test cases